src/unix/linux_like/linux/mod.rs

```diff
@@ -73,6 +73,14 @@ missing! {
     pub enum fpos64_t {} // FIXME: fill this out with a struct
 }
 
+cfg_if! {
+    if #[cfg(target_env = "musl")] {
+        pub type __itimer_which_t = ::c_int;
+    } else {
+        pub type __itimer_which_t = ::c_uint;
+    }
+}
```

Comment:
glibc has `int` when it is not defined as an enum, and musl doesn't seem to define this type at all - why is the definition done this way?

Reply:
Ah nevermind, I see the reasoning below. I think we need to work with one of the ways that doesn't expose this on musl.

In general it is probably better to avoid Rust `enum`s to represent C types at all, given how easy it is to cause UB. So some sort of solution that just typedefs an int on glibc seems preferable to me.